icaruscode/Timing/PMTBeamSignalNamespaceConverter_module.cc

```diff
@@ -0,0 +1,102 @@
+////////////////////////////////////////////////////////////////////////
+// Class:       PMTBeamSignalNameSpaceConverter_module
+// Plugin Type: producer (Unknown Unknown)
+// File:        PMTBeamSignalNameSpaceConverter_module.cc
+//
+// Generated at Thu Jun  5 14:59:32 2025 by Anna Heggestuen using cetskelgen
+// from cetlib version 3.18.02.
+////////////////////////////////////////////////////////////////////////
+#include "sbnobj/Common/PMT/Data/PMTBeamSignal.hh"
+#include "icaruscode/IcarusObj/Legacy/PMTBeamSignal.h"
+
+#include "art/Framework/Core/EDProducer.h"
+#include "art/Framework/Core/ModuleMacros.h"
+#include "art/Framework/Principal/Event.h"
+#include "art/Framework/Principal/Handle.h"
+#include "art/Framework/Principal/Run.h"
+#include "art/Framework/Principal/SubRun.h"
+#include "canvas/Utilities/InputTag.h"
+#include "fhiclcpp/ParameterSet.h"
+#include "messagefacility/MessageLogger/MessageLogger.h"
+
+#include <memory>
+
+namespace sbn::timing {
+  class PMTBeamSignalNamespaceConverter;
+}
```

Comment:
```suggestion
namespace icarus::timing {
  class PMTBeamSignalNamespaceConverter;
}
```
I would put this new producer under `icarus::timing` since it will be run only on the ICARUS side of things.